unikraft/app/application.go

```diff
@@ -347,6 +347,24 @@ func (app application) MakeArgs(tc target.Target) (*core.MakeArgs, error) {
 		unformattedLibraries[k] = v
 	}
 
+	// All known language libraries right now
```

Comment:
This comment seems a bit cryptic to me. What does it mean?

Reply:
I would suggest a fuller comment such as "Add language libraries that we know require a specific ordering. Currently, these are C++-related libraries. Other may be added as required."

Reply:
`usually sit` sounds very scary. Basically, if we do this there is no (easy) way to change the libraries ordering (placing a libc in front of a language library), right?

Is there a reason to change the library ordering from the `kraftkit` internals? Why not just go with the ordering from the `kraft.yaml` file? That sounds to me like the easiest way to go, leave some control to the user, do not hardcode library ordering from within kraftkit.

Reply:
@StefanJum , as in the PR comment: " This is a temporary fix as mentioned in the comment above them."

@craciunoiuc , I would add some "FIXME" or "TODO" item in front of the comment, together with the sentence "This is a temporary fix", as something to be handled properly in the future.

Reply:
there is a previous todo in there above
![image](https://github.com/unikraft/kraftkit/assets/46280801/51a9694c-6484-43ac-8d8e-6d928fb9b64a)

Reply:
@StefanJum because the libraries in the `kraft.yaml` are saved in a map so the order is not preserved